repair/repair.cc

```diff
@@ -1540,3 +1540,109 @@ future<> sync_data_with_repair(seastar::sharded<database>& db,
         });
     });
 }
+
+future<> bootstrap_with_repair(seastar::sharded<database>& db, locator::token_metadata tm, std::unordered_set<dht::token> tokens) {
```

Comment:
What is `tokens`? The joining node's tokens?

If so the name should indicate it.

Reply:
OK. It is the bootstrap_tokens.